js/web/lib/onnxjs/session.ts

```diff
@@ -60,7 +60,7 @@ export class Session {
       this._model = new Model();
       if (typeof arg === 'string') {
         const isOrtFormat = arg.endsWith('.ort');
-        if (typeof fetch === 'undefined') {
+        if (typeof process !== 'undefined' && process.versions && process.versions.node) {
```

Comment:
maybe we should have a isNodejs() function in utils.

Reply:
the 3 duplicated implementation are defined in lib/onnxjs, lib/wasm and test. they are separated targets that designed to not share common code.